pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -45,6 +45,11 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 	// loop until we find the secret necessary to connect to the external cluster
 	// then populate clusterInfo
 	cluster.Info = mon.PopulateExternalClusterInfo(c.context, c.namespacedName.Namespace)
+	if cluster.Info == nil {
```

Comment:
Will clusterInfo ever be nil here? I thought `PopulateExternalClusterInfo()` didn't return until it was available.